sigridci/sigridci.py

```diff
@@ -243,31 +243,34 @@ def __init__(self, options):
 
     def prepareUpload(self, sourceDir, outputFile):
         zipFile = zipfile.ZipFile(outputFile, "w", zipfile.ZIP_DEFLATED)
+        zipContents = []
         
         for root, dirs, files in os.walk(sourceDir):
             for file in sorted(files):
                 filePath = os.path.join(root, file)
                 if file != outputFile and not self.isExcluded(filePath):
                     relativePath = os.path.relpath(os.path.join(root, file), sourceDir)
                     uploadPath = self.getUploadFilePath(relativePath)
+                    zipContents.append(uploadPath)
```

Comment:
Correct. `os.walk` is weird in that it returns `root, dirs, files` as different lists, which is different from Java where you just get everything. Since we only iterate `files`, this indeed does not get called for directories.